softwareComponents/rust-rofi_voxel/src/voxel_world/world_rotation.rs

```diff
@@ -0,0 +1,158 @@
+use super::VoxelWorld;
+use crate::pos::VoxelPos;
+use crate::{atoms, voxel::VoxelBody};
+use enum_iterator::Sequence;
+
+#[derive(Debug, PartialEq, Eq, Sequence)]
+pub struct WorldRotation {
+    /// Axis at which rotated [1,0,0] has value 1
+    x_rotates_to: atoms::Axis,
+    /// Is axis negated before rotation
+    neg_axis: [bool; 3],
+}
```

Comment:
Bez komentáře se mi velmi špatně chápe, jak je rotace vyjádřena. Je to ta osa, která má skončit směrem nahoru?

Reply:
Tím jsem chtěl naznačit, že by tato informace mohla být zanesena v kódu.